consul_exporter.go

```diff
@@ -348,13 +355,20 @@ func main() {
 	}
 	prometheus.MustRegister(exporter)
 
+	queryOptionsJson, _ := json.Marshal(queryOptions)
```

Comment:
What about handling a potential error and printing a warning with log.Warnf instead of swallowing it silently?

Reply:
done